sssd_test_framework/utils/authentication.py

```diff
@@ -538,6 +538,54 @@ def password_expired(self, username: str, password: str, new_password: str) -> b
 
         return result.rc == 0
 
+    def password_with_output(self, username: str, password: str) -> tuple[int, str, str]:
```

Comment:
This is exactly the same as password, right? If yes, we should use it in the password method:

```python
def password(self, username: str, password: str) -> bool:
    rc, _, _ = self.password_with_output(username, password)
    return rc
```

Reply:
The only difference is setting log_user 1 in expect.